src/commands/poll.rs

```diff
@@ -37,9 +39,11 @@ pub async fn poll(ctx: &client::Context, msg: &Message, args: Args) -> CommandRe
 /// Have others select one of many options.
 #[command("multi")]
 #[usage("poll multi [title] <one option per line>")]
-async fn poll_multi(ctx: &client::Context, msg: &Message, args: Args) -> CommandResult {
-    let mut lines = args.rest().lines().collect_vec();
-    let title = lines.first().and_then(|line| line.strip_prefix("multi "));
+async fn poll_multi(ctx: &client::Context, msg: &Message) -> CommandResult {
+    let lines = msg.content.lines().collect_vec();
+    let title = lines.first().map(|line| line.split_at_word("multi").1);
+    let mut lines = lines.clone();
+
     if !lines.is_empty() {
         lines.remove(0);
     }
```

Comment:
Just realized this could be a tiiiiiiny bit more efficient if we do the following:
```suggestion
    let mut lines_iter = msg.content.lines();
    let title = lines.next().map(|line|line.split_at_word("multi").1);
    let options_lines = lines_iter.collect_vec();
```
and then change the rest of the code to use `options_lines`, obviously. This saves a single clone and makes it a biiiiiiiiiit cleaner I'd say? not much tho.

How it works is that by doing `.next()` we advance the iterator by one element, giving us the first line as an Option. Then, when we collect, the first line will have already been consumed and won't be included in the vec, thus we have skipped it automatically, pretty much. Please test this before tho, I'm just writing it in github rn :kek: